resolwe/storage/management/commands/compare_models_and_csv.py

```diff
@@ -0,0 +1,350 @@
+""".. Ignore pydocstyle D400.
+
+======================
+Compare models and CSV
+======================
+
+"""
+import csv
+import logging
+import os
+
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+from resolwe.flow.models import Data
+from resolwe.storage.models import ReferencedPath
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """Compare Django's Data records with a CSV file and log differences.
+
+    expected CSV format:
+        bucket_id, file_key, file_size, file_hash
+    """
+
+    help = (
+        "Log differences between Data records in models (Django database)"
+        " and records from a CSV file (S3 inventory)."
+        " Expected CSV format: bucket_id, file_key, file_size, file_hash"
+    )
+
+    def add_arguments(self, parser):
+        """Add command arguments."""
+        # Positional argument: path to CSV file
+        parser.add_argument("csv_path", nargs=1, type=str)
+
+    def handle(self, *args, **options):
+        """Command handle."""
+        CSV = FileIterator(options["csv_path"][0])
+        models = Data.objects.all().order_by("location__pk")
```

Comment:
Please use a more descriptive variable name. Name ``models`` is to generic. 
Maybe ``data_models`` ?